samples/openthread/ncp/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,13 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <1000000>;
+	status = "okay";
+};
+
+&uart1 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <1000000>;
+	status = "okay";
+};
```

Comment:
According to latest findings using CC310 as a RNG device gives a lot of delay. Test if switcing to default RNG device improves performance and apply needed changes.

Please note that CC310 is available only on *840 devices, not on 833. You may need to update this file if dts.overlay is ignored when this file is found otherwise you can use general dts.overlay file as in the PR below.

this PR should point you on how to do that:
https://github.com/nrfconnect/sdk-nrf/pull/2572/files

Reply:
Done